spec/models/poll_spec.rb

```diff
@@ -2,5 +2,63 @@
 
 RSpec.describe Poll, type: :model do
     it { should validate_presence_of(:title) }
-    it { should have_many(:candidates)}
+    it { should have_many(:candidates).dependent(:destroy) }
+    it { should accept_nested_attributes_for(:candidates) }
+
+    describe "#winning_candidates" do
+        let(:poll) { Poll.create!(title: "Poll", start_time: "2022-09-29T16:31", end_time: "2022-09-29T14:34") }
+        let(:candidate_one) { Candidate.create!(title: "Candidate one", poll: poll) }
+        let(:candidate_two) { Candidate.create!(title: "Candidate two", poll: poll) }
+        let(:candidate_two) { Candidate.create!(title: "Candidate three", poll: poll) }
+
+        subject { poll }
```

Comment:
The subject should always be the method under test when you are unit testing a method. In this case it would be `poll.winning_candidates`